requirements.txt

```diff
@@ -37,4 +37,5 @@ uvloop==0.14.0
 wcwidth==0.2.3
 websockets==8.1
 zipp==3.1.0
-pandas-profiling[notebook]
\ No newline at end of file
+pandas-profiling[notebook]
+streamlit==0.61.0
```

Comment:
You should add streamlit on Pipenv file.

Reply:
Done!